storagenode/console/satellites.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package console
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Satellites queries satellite related info from db
+type Satellites interface {
+	// GetIDs returns list of satelliteIDs that storagenode has interacted with
+	// at least once
+	GetIDs(ctx context.Context, from, to time.Time) (storj.NodeIDList, error)
```

Comment:
Why do we need a time range for this?

Reply:
Because ids is gathered from bandwidth table, selecting only distinct ones. If you change time window, your result list will also change

Reply:
Doesnt that mean that if i store data for satellite X and it didnt download or upload new data to me, it wont get selected? I think that is very sensitive to failures and/or miscalculations

Reply:
Yeah, let’s agree on all satellites, I have changed code to it already